sdk/scheduler/src/test/java/com/mesosphere/sdk/testutils/OfferRequirementTestUtils.java

```diff
@@ -47,11 +51,13 @@ public static DcosCluster getTestCluster(String version) {
     }
 
     public static SchedulerFlags getTestSchedulerFlags() {
```

Comment:
`SchedulerFlags` doesn't seem like a totally accurate ClassName anymore. I think having a minimal interface to the Scheduler's system/environment is sensible (e.g. not exposing a bunch of envvars to then only use them one place).

What about calling it `SchedulerEnvironment` or `SchedulerSystem`?

Reply:
How about `SchedulerConfig`? `SchedulerSystem` feels like it'd be invoking ulimits settings or something, while `SchedulerEnvironment` is a bit further into being specific to just envvar fetching.